lib/portage/tests/emerge/test_simple.py

```diff
@@ -294,6 +294,13 @@ def _async_test_simple(self, loop, playground, metadata_xml_files):
 			port=binhost_server.server_port,
 			path=binhost_remote_path)
 
+		binpkg_format = settings.get("BINPKG_FORMAT", "xpak")
```

Comment:
I think you'd want to have separate tests or test variants for both package formats, so both are tested all the time.